README.md

```diff
@@ -1 +1,86 @@
-# elixir-starting-guide
\ No newline at end of file
+# How to install Elixir with asdf?
+
+## Why is it important?
+- Elixir runs on the Erlang VM
+- If you install less-compatible versions of Erlang and Elixir you might
+run into problems when trying to run your code
+- The version of Erlang is often referred as the OTP version. The version of Elixir used must be compiled specifically for the version of Erlang in use.
+
+!["elixir -v"](/images/installation/elixir-v.png)
+
+- Note the first line Erlang/OTP 26 must match the Erlang version that Elixir was compiled against. If they do not match, you may encounter issues.
+- Knowing you can use `elixir -v` to see both versions helps you quickly tell when you have compatible versions being used together.
+
+## Why use asdf?
+- asdf is CLI tool which can manage multiple language runtimes - so you
+can install two matching versions of Erlang and Elixir to avoid the above
+mentioned problems
+
+## How to install asdf?
+- Follow the instructions [here]!(https://asdf-vm.com/guide/getting-started.html#_3-install-asdf)
+
+## Install Erlang
+- First we need to install the asdf plugin that installs Erlang and manages the versions for us.
+- The following command adds the Erlang plugin:
```

Comment:
We don't have something like this in nvm because it only handles node, that's why we need these plugins.